### DISCUSSION TITLE
Zwift Ride & Rouvy

### DISCUSSION THREAD

**Discussion by Thin-White-Duke** [cagnulein/qdomyos-zwift#2697] [Q&A]
Hello Roberto,
I'm quite exited when I heard about the possibilities of your app, esp. that I can use virtual shifting of my Zwift Ride together with Rouvy. I already downloaded the app on my iPhone, then I read (somewhere) that I have to use a beta-version to get things going. Is the current version that I downloaded yesterday from the App Store the correct one, or do I have to upgrade to the beta version somehow?

Thanks and congratulations to your awesome work!!!

**Comment by cagnulein** [comment]
That version is already perfect! Give it a try and let me know!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 25 ott 2024 alle 18:36 Thin White Duke <
***@***.***> ha scritto:

**Comment by Thin-White-Duke** [comment]
Hi Roberto,
I tried with my Zwift Ride Virtual shifting and Rouvy. In principal it works quite fine, but I also made some observations - maybe they are of interest for you (or they are simply a lack of my own competence in using your cool app appropriately 😊).
- I have the impression, that riding at the same slope, same gear, no draft does not feel like I have the same resistance on the pedals (unlike on Zwift courses where I have direct connection to the virtual shifters, w/o QZ)
- The increment when shifting gears is very different when using the +/- tiles on QZ compared to using the buttons on my Zwift play. E.g. with a gear gain of 1 the tiles move the gears by 1, the buttons shift gears more than one (2-3) and even the levers on the Zwift play shift the gears by another, different increment.
- The increment when shifting gears up is different than shifting down using the Zwift play buttons/levers
- It gets even more complicated when I use a real number for the gear ratio like 0.5. Shifting 3 times up, then 3 times down I'm not back at the same gear
- I tried the experimental feature to use the Zwift gears table logic. At start, I have the gear 0 shown, suppose this the minimum gear, at least I cannot decrease it to neg. values ... the resistance with this gear on flat ground is quite high, so when I enter in a section with some slope > 4% it's almost impossible to keep the wheels turning (at least for me). Is it possible to adjust/offset the resistance for gear 0 when using the Zwift gear logic? 
- I do not really understand the gears offset parameter in the general bike options (could this also be a negative number?). Does it also work in together with the Zwift gear logic?

Sorry for being that lengthy with my observations and descriptions.
Cheers!

**Reply by cagnulein** [reply]
> Hi Roberto,
> 
> I tried with my Zwift Ride Virtual shifting and Rouvy. In principal it works quite fine, but I also made some observations - maybe they are of interest for you (or they are simply a lack of my own competence in using your cool app appropriately 😊).

Great!

> 
> - I have the impression, that riding at the same slope, same gear, no draft does not feel like I have the same resistance on the pedals (unlike on Zwift courses where I have direct connection to the virtual shifters, w/o QZ)

If you can send me a debug log I can check it https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

> 
> - The increment when shifting gears is very different when using the +/- tiles on QZ compared to using the buttons on my Zwift play. E.g. with a gear gain of 1 the tiles move the gears by 1, the buttons shift gears more than one (2-3) and even the levers on the Zwift play shift the gears by another, different increment.

Did you enable the gear debouncing setting in the zwift accessories settings?

> 
> - The increment when shifting gears up is different than shifting down using the Zwift play buttons/levers

Did you play with the gear zwift experimental setting in the zwift accessories settings? Are you on android or ios?
> 
> - It gets even more complicated when I use a real number for the gear ratio like 0.5. Shifting 3 times up, then 3 times down I'm not back at the same gear

Maybe it's because of the slope? Anyway with a debug log everything will be clear 

> 
> - I tried the experimental feature to use the Zwift gears table logic. At start, I have the gear 0 shown, suppose this the minimum gear, at least I cannot decrease it to neg. values ... the resistance with this gear on flat ground is quite high, so when I enter in a section with some slope > 4% it's almost impossible to keep the wheels turning (at least for me). Is it possible to adjust/offset the resistance for gear 0 when using the Zwift gear logic? 

Share a debug log and I will understand this

> 
> - I do not really understand the gears offset parameter in the general bike options (could this also be a negative number?). Does it also work in together with the Zwift gear logic?

Do you mean gear gain?

> 
> 
> 
> Sorry for being that lengthy with my observations and descriptions.
> 
> Cheers!

**Reply by Thin-White-Duke** [reply]
Thank you very much!

> If you can send me a debug log I can check it https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

I will log my new session and send you the report.

> Did you enable the gear debouncing setting in the zwift accessories settings?

No, will do the next time.

> Did you play with the gear zwift experimental setting in the zwift accessories settings? Are you on android or iOS?

I will try the "Buttons debouncing" option for sure. I tried "Zwift app ratio for gears", but in the smallest gear "0" (could not switch to a lower/neg. gear, but I think this is intended so) the starting resistance is too high to pedal comfortable. I'm on iOS.

> Do you mean gear gain?

No, the option right below: "Gears Offset". I thought if I put that to e.g. -5 the gear "0" when using "Zwift app ratio for gears", than I can lower the resistance.

**Reply by cagnulein** [reply]
Ok let me know! With a debug log will be easier to answer to your questions!
